src/base/bittorrent/session.cpp

```diff
@@ -382,7 +382,10 @@ Session::Session(QObject *parent)
     , m_diskCacheTTL(BITTORRENT_SESSION_KEY(u"DiskCacheTTL"_qs), 60)
     , m_diskQueueSize(BITTORRENT_SESSION_KEY(u"DiskQueueSize"_qs), (1024 * 1024))
     , m_diskIOType(BITTORRENT_SESSION_KEY(u"DiskIOType"_qs), DiskIOType::Default)
-    , m_useOSCache(BITTORRENT_SESSION_KEY(u"UseOSCache"_qs), true)
+    , m_diskIOReadMode(BITTORRENT_SESSION_KEY(u"DiskIOReadMode"_qs), DiskIOReadMode::EnableOSCache
+        , clampValue(DiskIOReadMode::DisableOSCache, DiskIOReadMode::EnableOSCache))
```

Comment:
It seems those values are stored in enum type on disk, so I think there is no need to clamp the values (here and `m_diskIOWriteMode`). The underlying classes will verify the value and reset to default value if it is incompatible/invalid.
```suggestion
    , m_diskIOReadMode(BITTORRENT_SESSION_KEY(u"DiskIOReadMode"_qs), DiskIOReadMode::EnableOSCache)
```